### ISSUE TITLE
Community Votes display for imported Observations

### ISSUE THREAD

**Issue by JoeCohen** [MushroomObserver/mushroom-observer#2336]
- Community Votes are incorrectly displayed as `0%` for Observations imported in the background from iNat. 
- They are still incorrect after a page refresh
- The Community Vote for a only one Proposed Name is displayed correctly only after:
  - clicking that percentage, 
  - closing the modal, and 
  - refreshing the screen. 
 
See attached screenshots below.

@nimmolo, @nwilson-eol: 

### Questions

- Is there something I could do in the background to prevent this problem?
- Would it be possible to recalculate Community Votes when an imported Observation is first displayed?  (`observation.num_views.zero? && observation.source == "mo_inat_import"`)
  - I thought there was a method to recalculate Community Votes, but I can't find it.
  - I know this would slow down the initial display of the Obs, but it's only one time.
- Other suggestions welcome.

### To replicate 

(I think this will work for any user, but am not 100% sure):
- Checkout `jdc-1955-inat-import`
- start/restart SolidQueue (just to be sure it isn't the source of the problem)
- `rails lang:update; rails db:migrate; rails s`
- `http://localhost:3000/observations/inat_imports/new`
- Your iNat Username: `dannymi`
- Comma separated list of observation #s: `24970904`
- Check `(Required) I consent ...` checkbox
- Submit
Result: Observations by Recent Activity, without the new Obs.
- Wait 10 seconds (to ensure that the background creation of the Obs is complete).
- Refresh
Result: Observations by Recent Activity, including the new Obs.
- Click on the new Obs.
- Scroll to the Proposed Names panel
Result: 4 Proposed Names, each showing `(1)` vote, with Community Vote `0%`  
<img width="583" alt="Screenshot 2024-08-26 at 7 36 18 AM" src="https://github.com/user-attachments/assets/c302640d-bdff-4aa0-b603-c18ff4208da2">

- Refresh page
Result: No change

- Click `0%` next to _Calvatia_ (or any other Proposed Name)
Result: 
Modal shows correct Community Vote ("Overall score")
> 
```
I’d Call It That  3.0 4.05  1  (MO Webmaster)
...
Overall score: ~80.17%
``` 
<img width="611" alt="Screenshot 2024-08-26 at 7 39 09 AM" src="https://github.com/user-attachments/assets/93b2eb3b-f520-4240-8705-b447f154f96e">

- Close the modal
- Refresh the screen
Result: Community Vote `80%` for only the first Proposed Name (_Calvatia_) 
<img width="527" alt="Screenshot 2024-08-26 at 7 42 52 AM" src="https://github.com/user-attachments/assets/e2e86db2-7653-4618-a4a2-b7c65dd1d4f3">

- Repeat clicking the percentage for another name, closing the modal and refreshing.
Result: Each refresh fixes one Proposed Name.

**Comment by JoeCohen** [comment]
Figured it out.  I need to call `NamingConsensus::calc_consensus`

**Comment by JoeCohen** [comment]
Fixed in [2aa3331](https://github.com/MushroomObserver/mushroom-observer/pull/2150/commits/2aa3331ffae07337247acf7fccfd0a37eb0c4611)
